Dockerfile

```diff
@@ -9,13 +10,15 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     ca-certificates \
     git \
     openjdk-8-jdk \
-    python \
+    python3-dev \
     maven \
     wget \
     && rm -rf /var/lib/apt/lists/*
 
-RUN wget -q https://bootstrap.pypa.io/pip/3.5/get-pip.py \
-    && python get-pip.py pip==18.1 \
+RUN update-alternatives --install /usr/bin/python python /usr/bin/python3 10
+
+RUN wget -q https://bootstrap.pypa.io/get-pip.py \
```

Comment:
Now that we're not having to install an old version, we can install pip via apt-get, as per https://github.com/aodn/cloud-deploy/commit/2aa227aed035494efb3d0d4f6a025247baa01baa